src/js/device/DeviceCommands.js

```diff
@@ -629,10 +630,12 @@ export default class DeviceCommands {
     async getAccountDescriptor(
         coinInfo: CoinInfo,
         indexOrPath: number | number[],
+        derivationType: ?CardanoDerivationType,
```

Comment:
shouldn't this be required here and then we could simplify 663 and following lines?